### ISSUE TITLE
Stats: Apply `showNewModules` property to all instances of `StatsModule`

### ISSUE THREAD

**Issue by jsnmoon** [Automattic/wp-calypso#71638]
Apply the new horizontal bars to all instances of the `StatsModule` component:

- [x] On the traffic page
- ~~On the insights page. This page may require a new grid layout to accommodate the horizontal bars.~~ moved to #72038
- [x] On the individual stats modules pages (part of #71018).
    - [x] `/stats/day/posts/example.com`
    - [x] `/stats/day/referrers/example.com`
    - [x] `/stats/day/countryviews/example.com`
    - [x] `/stats/day/authors/example.com`
    - [x] `/stats/day/searchterms/example.com`
    - [x] `/stats/day/clicks/example.com`
    - ~~`/stats/day/videoplays/example.com`~~ moved to #71018
    - ~~`/stats/day/filedownloads/example.com`~~ moved to #71018

Identify and note any additional pages requiring migration in the above to-do list.

Once all of the `StatsModule` components use horizontal bars, we can remove the `showNewModule` property. It was used to prevent horizontal bars from showing on the `Insights` page.

cc @grzegorz-cp.

**Comment by grzegorz-cp** [comment]
I opened a [draft PR](https://github.com/Automattic/wp-calypso/pull/71822) for removing `showNewModule` property. Besides this, I added a feature flag controlling the bars - without the FF, it should only be visible on the Traffics page, but with the FF, it is visible everywhere.

This highlights more work that needs to be done, but I didn't want to inflate the above PR so it wouldn't stay open for a week.

The summary details page will be updated in another PR. First, I need to replace the card's title and update the page header (below breadcrumbs, where you switch periods). Existing "tabs" have to look like buttons next to the chart.

I'm holding off on applying the grid layout on the Insights page while investigating why specific tags (each tag, not a category) don't have any view value. So this change will come in yet another PR.

cc @jsnmoon @jeffgolenski

**Comment by grzegorz-cp** [comment]
I noticed that this touches on [#71018](https://github.com/Automattic/wp-calypso/issues/71018).
When I removed `showNewModule` in the PR mentioned above, I had to touch on the header for the summary details page. It shares styles with other headers, so I'll open another PR to add an updated header as part of this issue.
I can also add a grid, but I propose handling any tweaks to the header and summary pages as part of the Modernization project. The same would apply to adjusting modules placed on the grid on the Insights page.

**Comment by grzegorz-cp** [comment]
While working on the ticket, I discovered that the `Videos`, `File downloads` and `Authors` modules differ from the rest.
`Authors` and `File downloads` use the same component as the other modules to display the summary page (and they have horizontal bars now), but they don't show a header on the summary page. They are excluded because the header menu changing the period doesn't work (I guess the API call is different).
`Videos` use a completely different layout because it shows multiple values in one row - impressions, watching time and views.
Because of the differences, those issues are moved to #71018 for further investigation.

Adding a grid to the Insights page has also been split into its issue.

**Comment by jsnmoon** [comment]
@grzegorz-cp: Feel free to close this issue once we've shipped #71957 and we've enabled the feature flag on production 👍

**Comment by wilmersondasilva** [comment]
@grzegorz-cp, should this still be in the "In Review" column? Is there something that still needs review?

**Comment by grzegorz-cp** [comment]
Good call @wilmersondasilva. I'll put it back to "in progress". As part of the other issue improving the summary page, I'm adding a component to convert period buttons to a select list. I would only like to remove the feature flag once this is done (therefore, I didn't close this issue).
